### ISSUE TITLE
It seems 'exclude' doesn't work

### ISSUE THREAD

**Issue by AlexKlim** [pderichs/sublime_rubocop#28]
Hello. 

I've created .rubocop.yml : 
``` 
AllCops:
  Exclude:
    - 'db/schema.rb'
    - 'bin/**/*'
    - 'config/initializers/devise.rb'
    - 'config/initializers/simple_form*'
    - 'db/migrate/*'
```

But in my migration I see error: 
```
Method has too many lines.
```

How can I fix it ?

**Comment by pderichs** [comment]
I am not sure what you mean with "migration", but your problem might be related to #25 

But this is only related to the per file checks in the plugin - if you use "RuboCop: Check current project" you should be fine.

**Comment by AlexKlim** [comment]
Yep, you're right. The problem related to https://github.com/pderichs/sublime_rubocop/issues/25
> If rubocop is called with a single file, it ignores the .rubocop.yml.
